gym/envs/__init__.py

```diff
@@ -1,5 +1,9 @@
 from gym.envs.registration import registry, register, make, spec
 
+# To be able to create new-style properties
+class Env(object):
```

Comment:
I added a property for mode, but I removed it afterwards

I'll remove and resubmit.